src/Database/migrations/2024_02_16_081258_update_user_disks_name_constraint.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('user_disks', function (Blueprint $table) {
+            $sm = Schema::getConnection()->getDoctrineSchemaManager();
+            $indices = $sm->listTableIndexes('user_disks');
+            if (array_key_exists('user_disks_name_unique', $indices)) {
+                $table->dropUnique('user_disks_name_unique');
+                $table->unique(['name', 'user_id']);
+            }
```

Comment:
Why not just this?

```suggestion
            $table->dropUnique('user_disks_name_unique');
            $table->unique(['name', 'user_id']);
```

Maybe because the `unique()` is still deleted in the other migration?

Reply:
If you apply those changes and then want to migrate again, an error is thrown, because `user_disks_name_unique` does not exist any more.

Reply:
When does this happen exactly? A migration should only be run a single time.

Reply:
You're right. I probably have changed the file name, which is why it migrated the file again and threw the error. The shorter version is already pushed.